Sources/HipMobileUI/Resources/Strings.resx

```diff
@@ -505,6 +505,33 @@ Verantwortlich für den Betrieb: Dr. Simon Oberthür</value>
   <data name="SkipExhibit_Title" xml:space="preserve">
     <value>Sehenswürdigkeit überspringen</value>
   </data>
+  <data name="SettingsScreenView_FurtherSettings_Title" xml:space="preserve">
+    <value>Weitere Einstellungen</value>
+  </data>
+  <data name="SettingsScreenView_RemoveAllDownloadsButton_Text" xml:space="preserve">
+    <value>Entfernen</value>
+  </data>
+  <data name="SettingsScreenView_RemoveAllDownloadsPrompt_Confirm" xml:space="preserve">
+    <value>Entfernen</value>
+  </data>
+  <data name="SettingsScreenView_RemoveAllDownloadsPrompt_Question" xml:space="preserve">
+    <value>Möchten Sie wirklich alle Downloads entfernen? Hiermit geben Sie Speicherplatz frei. Bilder und Audio Dateien müssen aber bei Bedarf erneut runtergeladen werden.</value>
+  </data>
+  <data name="SettingsScreenView_RemoveAllDownloadsPrompt_Reject" xml:space="preserve">
+    <value>Abbrechen</value>
+  </data>
+  <data name="SettingsScreenView_RemoveAllDownloadsPrompt_Title" xml:space="preserve">
+    <value>Alle Downloads entfernen</value>
+  </data>
+  <data name="SettingsScreenView_RemoveAllDownloads_Text" xml:space="preserve">
+    <value>Alle Downloads entfernen (Nutzen Sie diese Option, um Speicherplatz freizugeben)</value>
+  </data>
+  <data name="ExhibitsOverviewView_EmptyList" xml:space="preserve">
+    <value>Die Liste ist leer. Laden Sie Inhalte runter, um loszulegen!</value>
```

Comment:
Instead of "Die Liste ist leer.", I'd say smth like "Keine Sehenswürdikeiten verfügbar." as the user otherwise probably does not know what  would be displayed. (exhibitsoverviewview has no headline)

And proper German is "herunter" instead of "runter"